test/helpers/field_enums.py

```diff
@@ -7,8 +7,14 @@ def select_column_names(cls):
         return [f'"{field.name}"' for field in cls]
 
     @classmethod
-    def select_query(cls, table):
-        return f"SELECT {', '.join(cls.select_column_names())} FROM {table}"
+    def select_query(cls, table, prefix=False):
+        columns = cls.select_column_names()
+        """ More complex queries might require disambiguation, eg. where two 'id' attributes are being referenced
+            - 'relevant_table.id' this prefix would solve it"""
+        if prefix:
+            for column in range(len(columns)):
+                columns[column] = f"{table}.{columns[column]}"
```

Comment:
nit: I think this can be simplified a bit

```suggestion
            columns = [f"{table}.{column}" for column in columns]
```